src/Integration.UnitTests/MefServices/VsSessionHostTests.cs

```diff
@@ -335,20 +335,7 @@ public void VsSessionHost_ResetBinding_ErrorInReadingBinding()
         [TestMethod]
         public void VsSessionHost_IServiceProvider_GetService()
         {
-            // Arrange
```

Comment:
The code that is left in this test checks non-local services are returned.

I'm not sure what all the setup in the old `Arrange` lines 339-342 was for; my guess is that at one point the host fetched some VS services in the constructor.